### ISSUE TITLE
[Question] How to preserve emotion tags during fine-tuning?

### ISSUE THREAD

**Issue by thucdangvan020999** [canopyai/Orpheus-TTS-0.1#256]
After fine-tuning the model on approximately 1 hour of audio data, the model appears to have forgotten how to properly handle emotion tags

Issue Description
- Training data: ~1 hour of audio
- Loss improvement: 5.0 → 3.5
- Problem: Model no longer processes emotion tags correctly
- Suspected cause: Catastrophic forgetting during fine-tuning
- Example : https://vocaroo.com/12uq70NWrpnt

**Comment by amuvarma13** [comment]
Catastrophic forgetting is the issue. 

You need to mix in a small amount of tagged data into your model. A very quick solution would be generate synthetic data and mix this as a small fraction of the dataset. 

Also you don't need to have very much of this kind of data as you are just trying to retain this ability not learn it.

**Comment by bowen20010220** [comment]
Do you fine-tune the pretrained model or ft model?

**Comment by amuvarma13** [comment]
If you want to keep the tags you can start with the finetuned model - unless you plan on adding enough samples of your own data with tags in which case use pt
